hnn_core/__init__.py

```diff
@@ -7,4 +7,4 @@
 from .cells_default import pyramidal, basket
 from .parallel_backends import MPIBackend, JoblibBackend
 
-__version__ = '0.4.dev0'
+__version__ = '0.4'
```

Comment:
if this is a release candidate, I would call it `0.4rc0` or some such thing ... just in the import, the docs can be 0.4. pypi won't let you overwrite a release, so you'll have to update the number every time you make a release. You can keep incrementing `0.4rc1`, `0.4rc2` etc., until you are ready for a final one ... which is `0.4`